k2/util/Logging.h

```diff
@@ -0,0 +1,32 @@
+//
+// Created by songmeixu (songmeixu@outlook.com) on 2020/8/3.
```

Comment:
Please follow the header style of other files.

Reply:
@songmeixu  And I think it's better if you declare your copyright under Xiaomi Corporation, as Dan and I did.

Reply:
Sorry, it's from my IDE old setting, and I forget it in some files. Changes have been made referring to the existing style.